docker-compose.yml

```diff
@@ -132,7 +132,7 @@ services:
           - ./.ddriot:/root/project/.riot
 
     localstack:
-        image: localstack/localstack:0.13.1
+        image: localstack/localstack:0.14.1
```

Comment:
Is this needed?

Reply:
This resolved some unit test failure locally, specifically tests that used localstacks publish batch functionality